ansible/playbooks/srv/roundcube.yml

```diff
@@ -0,0 +1 @@
+../service/roundcube.yml
```

Comment:
RoundCube seems to me as more of an end user application instead of a service. How about moving it to the `app.yml` playbook instead? That one is meant for user-facing services, basically.